interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props in react is the method that information is passed between the parent and child class components 
 
   Researched answer:
-
+When data is passed from one component to another in react they use whats called props which is short for properties 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: Display object management 
 
-  Researched answer:
+  Researched answer: Document Object Model
 
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: Object oriented programming is where the focus of development is centered around the pieces of tools needed as opposed to the functions needed to perform 
 
-  Researched answer:
+  Researched answer: The storage and manipulation of data is what separates object and functional programming. Data can be stored in object in Object oriented programming and not in functional programing 
 
 
 
 4. What is the difference between a Float and an Integer in Ruby?
 
-  Your answer:
+  Your answer: A float is a number with a decimal point and a integer is a whole number
 
-  Researched answer:
+  Researched answer: same
```

Comment:
That decimal is certainly distinctive. Did you notice anything interesting by either one?